lib/__tests__/cookieJar.spec.ts

```diff
@@ -1145,11 +1145,8 @@ describe('loose mode', () => {
 it('should fix issue #132', async () => {
   const cookieJar = new CookieJar()
   await expect(
-    cookieJar.setCookie(
-      // @ts-ignore
-      { key: 'x', value: 'y' },
-      'http://example.com/',
-    ),
+    // @ts-expect-error test case is explicitly testing invalid input
```

Comment:
We _could_ do an `as any` assertion, but I think it makes more sense to use `@ts-expect-error` here because we are explicitly trying to do something invalid.